container/kvm/initialisation.go

```diff
@@ -82,6 +74,24 @@ func ensureDependencies() error {
 	return nil
 }
 
+func getRequiredPackages(a string) []string {
+	var requiredPackages = []string{
+		// `qemu-kvm` must be installed before `libvirt-bin` on trusty. It appears
+		// that upstart doesn't reload libvirtd if installed after, and we see
+		// errors related to `qemu-kvm` not being installed.
+		"qemu-kvm",
+		"genisoimage",
+		"libvirt-bin",
+		"qemu-utils",
+	}
+	if a == arch.ARM64 {
+		// ARM64 doesn't support legacy BIOS so it requires Extensible Firmware
+		// Interface.
+		requiredPackages = append(requiredPackages, "qemu-efi")
```

Comment:
Are we sure that adding it to the end is OK given that we have to have 'qemu-kvm' needs to be before 'libvirt-bin'?

I'd also like to see the qemu packages in the initial list together, so at the start is fine.